src/services/application-analytics.service.ts

```
@@ -0,0 +1,879 @@
+    interval: number = 30000
+  ): () => void {
+    if (!this.subscribers.has(topic)) {
+      this.subscribers.set(topic, new Set());
+    }
+    this.subscribers.get(topic)!.add(callback);
+
+    const intervalId = setInterval(async () => {
+      try {
+        switch (topic) {
+          case 'performance_metrics':
+            const metricsResponse = await this.getRealtimeMetrics();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)